tools/autograd/gen_python_functions.py

```diff
@@ -769,7 +769,7 @@ def get_py_formal_arg(arg):
         default = None
         if arg.get('default') is not None:
             default = arg['default']
-            if default == 'nullptr' or default == 'nullopt' or default == '{}':
+            if default == 'nullptr' or default == 'nullopt' or default == 'c10::nullopt' or default == '{}':
```

Comment:
why do you need c10::nullopt?  where does that come from?

Reply:
I remember JIT was generating that (register_aten_ops) last week. It doesn't seem to be needed any more, so removed this.

Reply:
Ah, this was still needed. It's added in https://github.com/pytorch/pytorch/blob/b6edd7bbb4d0f79065ad6dac99b1a50575170039/aten/src/ATen/native_parse.py#L26

I will either revert this or find a different fix in the next PR.